generators/app/templates/documentation/index.js

```diff
@@ -0,0 +1,45 @@
+const config = require('../config');
+const schemas = require('./schemas');
+const paths = require('./paths');
+
+const port = config.common.api.port || 8080;
+
+module.exports = {
+  openapi: '3.0.1',
+  info: {
+    version: '0.1.0',
```

Comment:
is this project version? shouldn't we get this information from package json?

Reply:
Acording to the documentation, the version is the version of the OpenAPI document (which is distinct from the OpenAPI Specification version or the API implementation version).
What do you think this version should be? In the examples they use "1.0.1"

Reply:
its ok for now